lib/active_merchant/billing/gateways/elavon.rb

```diff
@@ -366,30 +366,48 @@ def add_line_items(xml, level_3_data)
         }
       end
 
-      def add_stored_credential(xml, options)
+      def add_stored_credential(xml, payment_method, options)
+        return unless options[:stored_credential]
+
         network_transaction_id = options.dig(:stored_credential, :network_transaction_id)
-        case
-        when network_transaction_id.nil?
-          return
-        when network_transaction_id.to_s.include?('|')
-          oar_data, ps2000_data = options[:stored_credential][:network_transaction_id].split('|')
-          xml.ssl_oar_data oar_data unless oar_data.nil? || oar_data.empty?
-          xml.ssl_ps2000_data ps2000_data unless ps2000_data.nil? || ps2000_data.empty?
-        when network_transaction_id.to_s.length > 22
-          xml.ssl_oar_data options.dig(:stored_credential, :network_transaction_id)
-        else
-          xml.ssl_ps2000_data options.dig(:stored_credential, :network_transaction_id)
+        xml.ssl_recurring_flag recurring_flag(options) if recurring_flag(options)
+        xml.ssl_par_value options[:par_value] if options[:par_value]
+        xml.ssl_association_token_data options[:association_token_data] if options[:association_token_data]
```

Comment:
In the ticket it mentions the sending the `ssl_association_token`, but we are sending `ssl_association_token_data` Is there any issue with that?

Reply:
I'm not entirely sure to be honest. I basically copied and pasted Davids past PR that he used for certification. Let me look at the docs and I'll get back to you

Reply:
Looking at their docs I'm thinking it was a type because only ssl_association_token_data comes up